### ISSUE TITLE
Training on Pendulum example using LREN model

### ISSUE THREAD

**Issue by xubo92** [xl402/Deep-Koopman#2]
Hi @xl402 

Thanks for your report on the deep learning of koopman operator! 

I tried to reproduce part of your experiments on using LREN model to learn for pendulum task. But I encountered the issue that the training loss is not decreasing in the first 40 epoches, and then the loss becomes really large at around 50 epoches. I guess that might has somthing to do with gradient explosion due to the recurrent structure of LREN. Have you ever had similar training situations like this? 

Thanks！

Best,
Xubo
